### DISCUSSION TITLE
Date of the last successful backup and reload button for the new Dashboard

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#542] [Q&A (Community Support)]
First of all, a big thumbs up to the developers (if there are any other than @nikosdion ) for the last Panopticon update.
Great job with the new dashboard, fixing old bugs, and adding new features. For now, everything is working as expected.
After last Panopticon update, I had some trouble connecting backups to Panopticon, but I solved that by adding one letter to the secret key to the endpoint webpage
![aa](https://github.com/akeeba/panopticon/assets/17553018/9a2c5049-7678-4076-8eb1-c161cb212525)
and Reload Jooma update information inside Panopticon
![Akeeba-Panopticon](https://github.com/akeeba/panopticon/assets/17553018/04c618cb-227b-4e82-92ad-d9c9cb240cc3)

I have two suggestion for the new Dashboard
1. Add reload buttons "Reload Joomla update information" and "Reload extension information" like on Table
2. Add date of the last successful backup

![Akeeba-Panopticon-backup-date](https://github.com/akeeba/panopticon/assets/17553018/4535c0c6-650c-4207-9a17-2c276a22ffc7)

**Comment by nikosdion** [comment]
Panopticon is mostly me doing the development. Other contributors do spit-polishing and accessibility improvements of the interface (such as @brianteeman), others do translations, etc.

The Dashboard lacks the reload buttons on purpose. @brianteeman created the first iteration with the express purpose of having a simpler, less cluttered interface. In my opinion, the Dashboard is a "client" view, whereas the Tabular view is the "admin" view. Different needs, different presentation modes.

Regarding the date of the last successful backup, it neither fits there, nor does it make sense. You want to see it to figure out if the backup is "too old". This is exactly what this icon shows you, with a maximum backup date you can configure. Remember, this is analogous to the backup out of date quickicon plugin in Joomla. If you want details, go into Site Information just like on Joomla you'd go into Components, Akeeba Backup, Manage Backups.
